src/bikeracks/__init__.py

```diff
@@ -50,7 +51,21 @@ def create_app(test_config=None):
         os.makedirs(app.instance_path)
     except OSError:
         pass
-        
+    
+    # Create error logging file
+    if not app.debug:
+        if not os.path.exists('instance/logs'):
+            os.mkdir('instance/logs')
+        file_handler = RotatingFileHandler('instance/logs/errors.log', maxBytes=10240,
```

Comment:
This is pretty small actually, not even 10k? I would go bigger and fewer. Like 512k  and backupCount=5 or something.